examples/grafana-dashboard/main.tf

```diff
@@ -0,0 +1,22 @@
+resource "chronosphere_grafana_dashboard" "my_dashboard" {
```

Comment:
since we call them classic dashboards, should we drop the support for the old name here? cc @bg451